routes/article.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const router = express.Router()
+
+router.get('articles/article/:id', (req, res) => {
+    const db = req.app.get('db')
+    db.query('SELECT * FROM article WHERE id = ' + req.params.id, (error, results) => {
+
```

Comment:
Nitpick: I dont think we need this line.
```suggestion
```

Reply:
Are you referring to the error throw?

Reply:
No, the empty line.